src/components/loader/Loader.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { Box } from "../box";
+import { Flex } from "../container";
+import { Spinner } from "../spinner";
+import { Text } from "../text";
+import styled, { css } from "styled-components";
+import theme from "../../theme";
+
+export interface LoaderProps {
+  inContent?: boolean;
+  bottom?: boolean;
```

Comment:
it will be better to add position attribute: `position?: "top-left" | "bottom-left" | "top-right" | "bottom-right"`